### ISSUE TITLE
Item loses Remove Animation properties when removed + undo

### ISSUE THREAD

**Issue by kxlow** [h6ah4i/android-advancedrecyclerview#262]
**Version of the Advanced RecyclerView library:** 0.9.1

**What happens?** When an item is swiped to removed and the removal is undone, the item no longer has remove animations when deleting an item (without swiping)

**What is the expected behavior?** Item should retain animation.

**How to reproduce?**
Example project, only works with **Swipeable (Basic)** because too much work.
1. Go into Swipe/Swipeable Basic
2. Swipe away an item.
3. Undo the swipe.
4. Hit "New Button" at the top of the list. Note that all other items fade when disappearing, except the item you swiped; it will disappear all at once without fading.

Add animations are retained (working properly), but that's not demoed in the example project.

**Comment by h6ah4i** [comment]
Thanks your the issue report. The fix will be included in the next release 👍 

---

FYI: Try the fixed version of the library through JitPack:

```
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:07303f76fa0d81ecd5e216de7b58331a618becb8'
}
```

**Comment by h6ah4i** [comment]
@kxlow Hi. I have just released v0.9.2 and it includes the fix of this issue 😉
